projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1578,6 +1582,14 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     public handleClosed() {
         this.onClosed.emit();
     }
+
+    /**
+     * @hidden @internal
+     */
+    public toggleCaseSensitive() {
+        this.filteringOptions.caseSensitive = !this.filteringOptions.caseSensitive;
+        this.filteringPipeTrigger++;
+    }
```

Comment:
```suggestion
        this.filteringOptions = { caseSensitive: !this.filteringOptions.caseSensitive };
```
Changing the reference to the `filteringOptions` object will trigger the pipe. No need for an additional parameter in this case.

Reply:
Done as suggested